src/vl-tooltip.js

```diff
@@ -253,6 +251,14 @@
         default:
       }
     }
+    
+    // supplement bin from fieldDef, user should never have to provide bin
+    if (fieldDef.bin) {
+      supplementedFieldOption.bin = true;
+      supplementedFieldOption.formatType = 'string'; // we show bin range as string (e.g. "5-10")
```

Comment:
Did you mean it's inconsistent because we are showing `string` for binned fields while the rest of Vega is showing `number` for bin?

Tooltip will show bin range (e.g. `"6-7"`, `"15-25"`), so I set `formatType` to `string`. Setting `formatType` to `string` is not absolutely required. Tooltip will do auto type-inference and we will get the same results. I can remove this line if that helps.